.github/scripts/run-config.py

```diff
@@ -130,6 +130,52 @@ def run_bmconfig(config: BenchmarkModelConfig, repo_path: Path, output_path: Pat
         return
     subprocess.check_call(cmd, cwd=repo_path)
 
+def run_bmconfig_profiling(config: BenchmarkModelConfig, repo_path: Path, output_path: Path, dryrun=False):
+    nsys_path = "/opt/nvidia/nsight-systems/2022.2.1/bin/nsys"
+    profiling_cmd = [nsys_path, "profile", "-o", "", "-f", "true", "-c", "cudaProfilerApi", sys.executable, "run_sweep.py", "-d", config.device, "-t", config.test, "--is-profiling"]
```

Comment:
`run_sweep.py` is the benchmark script? How about splitting the related arguments into another command such as run_sweep_cmd?  And then concat them when subprocess.run

Reply:
Done.